Formula/postgresql.rb

```diff
@@ -1,14 +1,14 @@
 class Postgresql < Formula
   desc "Object-relational database system"
   homepage "https://www.postgresql.org/"
-  url "https://ftp.postgresql.org/pub/source/v12.4/postgresql-12.4.tar.bz2"
-  sha256 "bee93fbe2c32f59419cb162bcc0145c58da9a8644ee154a30b9a5ce47de606cc"
+  url "https://ftp.postgresql.org/pub/source/v13.0/postgresql-13.0.tar.bz2"
+  sha256 "80e750be8d436b54197636a02636f8fd3263ba6779bf865b04832495ea592296"
   license "PostgreSQL"
   head "https://github.com/postgres/postgres.git"
 
   livecheck do
     url "https://www.postgresql.org/docs/current/static/release.html"
-    regex(/Release v?(\d+(?:\.\d+)+)/i)
+    regex(/Release v?(\d+(?:\.\d+)*)/i)
```

Comment:
@samford or @nandahkrishna, comments?

Reply:
They have releases such as `12`, `12.1`, `12.2`, etc. and now, `13`. So I think we should be using `*` rather than `+`.

Reply:
Ah. Yes. Ofcourse.